core/src/components/searchbar/test/basic/searchbar.e2e.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('searchbar: basic', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(`/src/components/searchbar/test/basic`);
+  });
+
+  test('should not have visual regressions', async ({ page }) => {
+    await page.setIonViewport();
+    expect(await page.screenshot()).toMatchSnapshot(`searchbar-diff-${page.getSnapshotSettings()}.png`);
+  });
+
+  test('should show cancel button on focus if show-cancel-button=focus', async ({ page }) => {
+    const searchbar = page.locator('#basic');
+
+    await searchbar.evaluate((el: HTMLIonSearchbarElement) => el.setFocus());
+    await page.waitForChanges();
+
+    expect(searchbar).toHaveClass(/searchbar-has-focus/);
+    expect(await searchbar.screenshot()).toMatchSnapshot(
```

Comment:
Rather than take a screenshot can we check to see if the clear button is visible or not?

Reply:
Maybe one of these tests can ensure that the clear button is rendered correctly/has the correct styles, but I don't think both tests need screenshots.